docs/custom_theme/404.html

```diff
@@ -0,0 +1,18 @@
+{% extends "main.html" %}
+
+<!-- Content block -->
+{% block content %}
+
+<p style="text-align: center;font-size: 3em"><i class="md-icon">account_balance</i></p>
+<h1 id="404-page-not-found" style="text-align: center">404</h1>
+<p style="text-align: center"><strong>Deity not found in the Pantheon</strong></p>
+<p style="text-align: center">
```

Comment:
_Deity not found in the Pantheon_ doesn't really make sense to me? I think it relies on the user knowing both whay deity and Pantheon mean and then making the leap that we're using Pantheon with a different meaning to our product. Maybe I'm too literal but would _Page not found in the Pantheon documentation_ be clearer?

Reply:
As discussed, we can afford some humour here.